### PR TITLE
Fix a couple of things in CHANGELOG

### PR DESCRIPTION
**Pull request by drpatelh**
Many thanks to contributing to nf-core/tools!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/tools/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by drpatelh** [comment]
Ill make the changes you suggested in the PR to `master` here @MaxUlysse . Wrestling with the `nanoseq` pipeline at the moment but Ill come back to this. Thanks!

**Comment by drpatelh** [comment]
Added a line 👍 Let me know if its ok. 

Now to figuring out why these tests are failing 😅

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## v1.8
 
+### Continuous integration
+
+* GitHub Actions CI workflows are now included in the template pipeline
+* Travis CI tests will most likely be deprecated from the next `tools` release
```
Comment (ewels) [inline CHANGELOG.md]:
```suggestion
* Travis CI tests will be deprecated from the next `tools` release
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ## v1.8
 
+### Continuous integration
+
+* GitHub Actions CI workflows are now included in the template pipeline
```
Comment (ewels) [inline CHANGELOG.md]:
```suggestion
* GitHub Actions CI workflows are now included in the template pipeline
  * Please update these files to match the existing tests that you have in `.travis.yml`
```

I wonder if we should add some `TODO` statements in the CI workflow. Then the linting will throw a warning if it's not customised.

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,20 +21,20 @@
...
 ### Template pipeline
 
 * Fixed incorrect paths in iGenomes config as described in issue [#418](https://github.com/nf-core/tools/issues/418)
-* Fixed [incorrect usage of non-existant parameter](https://github.com/nf-core/tools/issues/446) in the template
+* Fixed incorrect usage of non-existent parameter in the template [#446](https://github.com/nf-core/tools/issues/446)
 * Add UCSC genomes to `igenomes.config` and add paths to all genome indices
 * Change `maxMultiqcEmailFileSize` parameter to `max_multiqc_email_size`
 * Export conda environment in Docker file [#349](https://github.com/nf-core/tools/issues/349)
-* Change remaining parameters from `camelCase` to `snakeCase` [#39](https://github.com/nf-core/hic/issues/39)
+* Change remaining parameters from `camelCase` to `snake_case` [#39](https://github.com/nf-core/hic/issues/39)
```
Comment (ewels) [inline CHANGELOG.md]: 😆

File: docs/lint_errors.md [hunk 4]
```diff
@@ -12,9 +12,9 @@ The lint test looks for the following required files:
   * The main nextflow config file
 * `Dockerfile`
   * A docker build script to generate a docker image with the required software
-* Continuous integration tests with either [GitHub actions](https://github.com/features/actions), [Travis CI](https://travis-ci.com/) or [Circle CI](https://circleci.com/)
+* Continuous integration tests with either [GitHub actions](https://github.com/features/actions) or [Travis CI](https://travis-ci.com/)
   * GitHub actions workflows for CI (`.github/workflows/ci.yml`), branch protection (`.github/workflows/branch.yml`) and linting (`.github/workflows/linting.yml`)
-  * Alternatively, `.travis.yml` or `.circleci/config.yml` continuous testing are still allowed but might be deprecated
+  * Alternatively, `.travis.yml` continuous testing is still allowed but may be deprecated in the near future
```
Comment (ewels) [inline docs/lint_errors.md]:
```suggestion
  * Alternatively, `.travis.yml` continuous testing is still allowed but will be deprecated in the near future
```

File: CHANGELOG.md (hunk 1 above)
Comment (drpatelh) [reply CHANGELOG.md]: Didnt want to shoot us in the foot! Ill change that 👍

File: CHANGELOG.md (hunk 2 above)
Comment (ewels) [reply CHANGELOG.md]: https://github.com/nf-core/tools/pull/488

File: CHANGELOG.md (hunk 1 above)
Comment (drpatelh) [reply CHANGELOG.md]: https://github.com/nf-core/tools/pull/483/commits/95f032133fceeb88af2ed73c94dac45bc36a63d7

File: docs/lint_errors.md (hunk 4 above)
Comment (drpatelh) [reply docs/lint_errors.md]: https://github.com/nf-core/tools/pull/483/commits/d4995b819c69f93b067f08e874b94342ce264c3d

File: CHANGELOG.md (hunk 2 above)
Comment (drpatelh) [reply CHANGELOG.md]: Ive committed a fix so when tests pass Ill merge 👍

**Pull request merged**